glean-core/src/upload/directory.rs

```diff
@@ -357,7 +392,7 @@ mod test {
         let directory_manager = PingDirectoryManager::new(dir.path());
 
         // Verify that processing the directory didn't panic
-        let data = directory_manager.process_dirs();
+        let data = directory_manager.process_dirs().1;
```

Comment:
nit: I hate how un-descriptive index based access is (`.1`). Not saying we need to change it but it requires me to go and look up the process_dirs() to understand what `.1` vs. `.0` means here

Reply:
me too. I don't like how we need to thread through the value here now. If we do want this then I might re-do that part.

Reply:
I'm all for adding this as a guardrail to keep /pending-pings in check, for what it's worth.